definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -6,10 +6,12 @@
 - Electricity Storage Type:
 
   - Energy Storage System:
-      description: Mainly to refer to the batteries
+      description: all types of energy storage including pumped hydro and batteries
+  - Energy Storage System|Pumped Hydro Storage:
```

Comment:
if we want to include all kind of storages as defined above, we should also add 
Energy Storage System|Hydro|Reservoir, as this also is another kind of storage. It usually does not take electricity from the grid to store it in a specific form but it could (see below) and it is a way to store electricity.
In the modelling we are using, which is quite rough, Reservoir units do not have pumping assets, but in real life there exist some 'Reservoir' kind of units with also Pumping assets. The difference is mainly the size of the reservoir ('Reservoir' is for big size reservoirs, which are managed by electricity system operators on a yearly base, whlle 'Pumped Storage' units are of lower size and thus are managed more on a weekly or day/night base.
Anyway, in the present state of the needs, we do not need the specific 'storage system' variables for the 'Reservoir' so this could wait.......

Reply:
I also noticed that in 'Electricity input' it is mentioned that 'Reservoir' can have pumps

Reply:
The tag *Energy Storage Types* is used for the just-added variables for injected and discharged energy into storage. In my opinion, if it is natural inflow into a reservoir, this is not discharged energy (in a storage sense) but simply electricity generation optimized over time, hence just an electricity-generation input.

And yes, I see that there is a note in the definition of hydro-reservoir that that it can have pumps, but I do not understand why this was added there in the first place. I vaguely recall that you and/or @erikfilias wanted to insert that a while ago.

Reply:
@danielhuppmann I both agree and disagree. yes 'Reservoir' is exactly what you mention, in the way we are using it now (with the data from the scenarios...); But in the real life 'Reservoir' is acting alltogether as the 2 concepts. It is electricity generation optimized over time, and it also is (for 'Reservoir' with pumps) discharged energy in storage sense. So if we want to be strictly following the definition, we should add it as a storage option (maybe with a note telling it is the case ONLY for reservoirs with pumping capacity); 
But now we do not need that (at least for the case studies that I know), we are not using any data with a detailed modelling of hydro such that it includes the disaggrgated reservoirs with also their pumping facilities (and in the main detailed modellings that are used for mid to long term horizons, even when disaggregated, pumping capacities are usually treated separately from 'reservoir' capacities). In the case we would like to run case studies with detailed 'daily like' operation optimisation then we would need to include reservoirs with pumping. 
But as I already said, we can keep that for a later stage and not bother with it now.